src/styles/index.scss

```diff
@@ -1,3 +1,74 @@
 body {
   margin: 0;
 }
+
```

Comment:
The file is named `index.scss`, but according to the task requirements, it should be `main.scss`. Please rename the file and ensure it is linked correctly in the HTML.